src/sentry/api/serializers/models/filechange.py

```diff
@@ -13,22 +13,27 @@ def get_attrs(self, item_list, user):
         commits = Commit.objects.filter(id__in=[f.commit_id for f in item_list]).select_related('author')
         author_objs = get_users_for_commits(commits)
         commits_by_id = {commit.id: commit for commit in commits}
+
+        repo_names_by_id = dict(Repository.objects.filter(id__in=[commit.repository_id for commit in commits]).values_list('id', 'name'))
```

Comment:
not necessary to change, but for long lines like this, i think we try to do something like:
```python
repo_names_by_id = dict(Repository.objects.filter(
    id__in=[commit.repository_id for commit in commits],
).values_list('id', 'name'))
```
to make it more readable